pkgs/by-name/su/surreal-engine/package.nix

```
@@ -0,0 +1,71 @@
+{
+  alsa-lib,
+  cmake,
+  dbus,
+  fetchFromGitHub,
+  lib,
+  makeWrapper,
+  openal,
+  pkg-config,
+  SDL2,
+  stdenv,
+  vulkan-loader,
+  waylandpp,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "surreal-engine";
+  version = "2024-08-31";
+
+  src = fetchFromGitHub {
+    owner = "dpjudas";
+    repo = "SurrealEngine";
+    rev = "784a2707fba5dc641bee85cc48e7e15dc7585044";
+    hash = "sha256-7k7/G+qIIVCxKZQUtQ31MODdN4bU2bP1PYgv6r6laUE=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+    pkg-config
+  ];
+  buildInputs = [
+    alsa-lib
+    dbus
+    openal
+    SDL2
+    vulkan-loader
+    waylandpp
+  ];
+
+  cmakeBuildType = "Release";
+  parallelBuild = true;
```

I don't think `parallelBuild` is a thing in regular `stdenv.mkDerivation`. Did you mean:

```suggestion
  enableParallelBuilding = true;
```

? That gets auto-enabled by using `cmake`, so this is redundant.

```suggestion
```